config/locales/de.yml

```diff
@@ -341,6 +342,7 @@ de:
         one: 'Konnte %{resource} nicht speichern: ein Fehler.'
         other: 'Konnte %{resource} nicht speichern: %{count} Fehler.'
   entry_year: Einreisejahr
+  find_client: KlientIn finden
```

Comment:
Why not simply "Klient"? 👿 

Food for thought: http://www.rhetorik.ch/Feministisch/Feministisch.html